sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -311,10 +313,12 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
       AlterNamespaceSetPropertiesExec(catalog.asNamespaceCatalog, ns, properties) :: Nil
 
     case SetNamespaceLocation(ResolvedNamespace(catalog, ns), location) =>
+      val nsPath = CatalogUtils.makeQualifiedNamespacePath(
+        CatalogUtils.stringToURI(location), conf.warehousePath, session.sharedState.hadoopConf)
```

Comment:
This will now qualify the namespace location for v2 command.

Reply:
`session.conf` will contain the qualified warehouse path, but using SQLConf is fine since it will be qualified anyway (same as v1):
https://github.com/apache/spark/blob/c1029dcff103b1044a55b728c6c98113a255de1e/sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala#L89